components/eventing-controller/pkg/backend/cleaner/jetstream.go

```diff
@@ -1,6 +1,16 @@
 package cleaner
 
-import "github.com/kyma-project/kyma/components/eventing-controller/logger"
+import (
+	"regexp"
+
+	"github.com/kyma-project/kyma/components/eventing-controller/logger"
+)
+
+var (
+	invalidSourceSegment = regexp.MustCompile(`[\s.>*]`)
+
+	invalidEventTypeSegment = regexp.MustCompile(`[\s>*]`)
```

Comment:
(nit) To me, segments means a whole block of characters. Maybe the name `invalidSourceCharacters` and `invalidEventTypeCharacters`?

Reply:
done!